### DISCUSSION TITLE
Can't click ModalOverlay to close an open Combobox

### DISCUSSION THREAD

**Discussion by elilabes** [adobe/react-spectrum#5337] [Q&A]
I have modal that has a number of Comboboxes in them. I'd like to make it so that when you click anywhere not in the Combobox it closes the Combobox. Currently it seems, I have to click inside the modal content area to close the Combobox.

I am not sure if I am missing some documentation or if this is expected (i'd hope not, it is a real pain).

Here is a rough example using React Aria Components' demo. https://codesandbox.io/s/react-typescript-forked-2ly428?file=/src/App.tsx

**Comment by snowystinger** [comment]
That looks like a bug. Thanks for the codesandbox demonstrating the issue <3

**Reply by elilabes** [reply]
Oh cool! Thanks for the quick reply - shall I log it as a bug report instead?

**Reply by snowystinger** [reply]
Took care of it for you :) https://github.com/adobe/react-spectrum/issues/5338
